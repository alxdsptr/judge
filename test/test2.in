"Hello"
(define x 42)
x
(define y x)
y
)
(? 1 2)
(+ x y)
(exit 0)