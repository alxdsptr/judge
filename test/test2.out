>>> "Hello"
>>> ()
>>> 42
>>> ()
>>> 42
Error
Error
>>> 84
exit
